Formula/testkube.rb

```diff
@@ -0,0 +1,36 @@
+class Testkube < Formula
+  desc "☸️ Kubernetes-native framework for test definition and execution"
+  homepage "https://testkube.io"
+  url "https://github.com/kubeshop/testkube/archive/refs/tags/v0.10.1.tar.gz"
+  sha256 "ddcc802c4150bf456dcc3c4b0a3323675ad41e4b200908d4eec23aadb27063d1"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    user = "brew"
+    tracking_id = "UA-204665550-8"
+    ENV["CGO_ENABLED"] = "0"
+    ldflags = %W[
+      -s -w
+      -X main.version=#{version}
+      -X main.builtBy=#{user}
+      -X github.com/kubeshop/testkube/pkg/analytics.testkubeTrackingID=#{tracking_id}
```

Comment:
Why would we add a tracking ID?

Reply:
Hi @SMillerDev we use this Tracking ID to gather **anonymous** data about the users such has, number of installations, sessions, etc... 
Is it not common to have the tracking id configured in this manner ?

Reply:
It's not common to have tracking defined in brew, no. If you need Google Analytics in your products, why not code it in the project itself?

Reply:
We don't want it to be hardcoded. We want to show our users, that we use it and they can change or remove it

Reply:
Except they can't really change it in brew since we don't support people building from source.

Reply:
I don't think users are meant to change this value. This ID is meant to tell Google what the analytics data is of, exactly. This is just like what we use in `brew`:
https://github.com/Homebrew/brew/blob/6b9fc226fe77bbe88bfe6be9d89eb4aa15f7a430/Library/Homebrew/utils/analytics.sh#L64-L71

Reply:
Thank you @SMillerDev @carlocab for your review. We will make the necessary changes shortly.